### PR TITLE
Add eslint with custom Pebble rules

### PR DESCRIPTION
**Pull request by franc0is**

### CODE REVIEW

**Comment by franc0is** [comment]
Should be ready to go

**Comment by franc0is** [comment]
Ah, tests aren't lint clean yet. stand by.

**Comment by franc0is** [comment]
Roger roger

File: examples/common/js/pbl_graphics.js [hunk 1]
```diff
@@ -1,208 +0,0 @@
-var Module = Rocky.Module();
```
Comment (franc0is) [inline examples/common/js/pbl_graphics.js]: This file isn't used anywhere and is stale

File: examples/tictoc/js/rocky-extended.js [hunk 2]
```diff
@@ -1,73 +1,79 @@
-if (typeof(Rocky) == "undefined") {
-    Rocky = {};
+/*eslint "no-unused-vars": [2, { "args": "none", "vars": "local" }]*/
```
Comment (franc0is) [inline examples/tictoc/js/rocky-extended.js]: I'm not sure this is the right thing to do here. It complains about TimerService and Window being unused

File: .gitignore [hunk 3]
```diff
@@ -3,3 +3,4 @@
 node_modules
 .cache
 build
+.grunt/*
```
Comment (HBehrens) [inline .gitignore]: :+1:

**Comment by HBehrens** [comment]
That's a great addition! Thank you :) Please merge in master before landing, you should have conflicts at least in the tictoc example.

**Pull request merged**